src/test/java/org/alfresco/rest/api/search/ResultMapperTests.java

```diff
@@ -79,6 +79,8 @@
 import org.alfresco.rest.api.search.impl.StoreMapper;
 import org.alfresco.rest.api.search.model.HighlightEntry;
 import org.alfresco.rest.api.search.model.SearchQuery;
+import org.alfresco.rest.api.search.model.SearchSQLQuery;
```

Comment:
Can we tidy up unused imports please? SearchSQLQuery and Assert do not seem to be used.